src/current/v24.3/manage-logical-data-replication.md

```diff
@@ -0,0 +1,199 @@
+  public           | vehicles                   | table | root  |                 153 | NULL
+(7 rows)
+~~~
+
+The schema for the `movr.crdb_replication.dlq_113_public_promo_codes` DLQ:
+
+~~~sql
+CREATE TABLE crdb_replication.dlq_113_public_promo_codes (
+    id INT8 NOT NULL DEFAULT unique_rowid(),
+    ingestion_job_id INT8 NOT NULL,
+    table_id INT8 NOT NULL,
+    dlq_timestamp TIMESTAMPTZ NOT NULL DEFAULT now():::TIMESTAMPTZ,
+    dlq_reason STRING NOT NULL,
+    mutation_type crdb_replication.mutation_type NULL,
+    key_value_bytes BYTES NOT VISIBLE NOT NULL,
+    incoming_row JSONB NULL,
+    crdb_internal_dlq_timestamp_id_ingestion_job_id_shard_16 INT8 NOT VISIBLE NOT NULL AS (mod(fnv32(md5(crdb_internal.datums_to_bytes(dlq_timestamp, id, ingestion_job_id))), 16:::INT8)) VIRTUAL,
+CONSTRAINT dlq_113_public_promo_codes_pkey PRIMARY KEY (ingestion_job_id ASC, dlq_timestamp ASC, id ASC) USING HASH WITH (bucket_count=16)
+)
+~~~
+
+## Schema changes
+
+When you start LDR on a table, the job will lock the schema, which will prevent any accidental [schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}) that would cause issues for LDR. There are some [supported schema changes](#supported-schema-changes) that you can perform on a replicating table, otherwise it is necessary to stop LDR in order to [coordinate the schema change](#coordinate-other-schema-changes).
+
+### Supported schema changes
+
+There are some supported schema changes, which you can perform during LDR **without** restarting the job:
+
+Allowlist schema change | Exceptions
+-------------------+-----------
+[`CREATE INDEX`]({% link {{ page.version.version }}/create-index.md %}) | <ul><li>Hash-sharded indexes</li><li>Indexes with a computed column</li><li>Partial indexes</li></ul>
+[`DROP INDEX`]({% link {{ page.version.version }}/drop-index.md %}) | N/A
+[Zone configuration]({% link {{ page.version.version }}/show-zone-configurations.md %}) changes | N/A
+[`ALTER TABLE ... CONFIGURE ZONE`]({% link {{ page.version.version }}/alter-table.md %}#configure-zone) | N/A
+[`ALTER TABLE ... SET/RESET {TTL storage parameters}`]({% link {{ page.version.version }}/row-level-ttl.md %}#ttl-storage-parameters) | <ul><li>`ALTER TABLE SET (ttl_expire_after = "")`</li><li>`ALTER TABLE RESET (ttl_expire_after = "")`</li><li>`ALTER TABLE RESET (ttl)`</li></ul>
+
+{{site.data.alerts.callout_danger}}
+LDR will **not** replicate the allowlist schema changes to the destination table. Therefore, you must perform the schema change carefully on both the source and destination cluster.
+{{site.data.alerts.end}}
+
+### Coordinate other schema changes
+
+To perform any other schema change on the table, use the following approach to redirect application traffic to one cluster. You'll need to drop the existing LDR jobs, perform the schema change, and start new LDR jobs, which will require a full initial scan.
```

Comment:
once jeff lands and backports some changes, the user will be able to restart the stream with a backup-restore flow.  we could make revisions here once jeff's work lands.

Reply:
I was thinking that we could just document the schema change bit w initial scan to be safer, as opposed to the backup/restore option, if you think b/r is safe too though then sure we can add

Reply:
I haven't changed anything here, because I wasn't sure there was a definite ask, so let me know if I do need to do anything.

Reply:
i no longer think we should doc b/r here.